qiime2/sdk/action.py

```diff
@@ -276,6 +276,45 @@ def _build_annotations(self):
 
         return annotations
 
+    def _build_numpydoc(self):
+        numpydoc = []
+        numpydoc.append(textwrap.fill(self.name, width=79))
```

Comment:
the width should actually be `75` and for parameter/returns descriptions I think they wrap at `71` because they're indented.